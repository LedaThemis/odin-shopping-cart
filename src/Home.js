import './Home.css';

function Home() {
  return (
    <div className="Home">
      <h2 className="welcome--header">Welcome to Our Shop!</h2>
      <p>We Hope you find what you're looking for.</p>
    </div>
  );
}

export default Home;
