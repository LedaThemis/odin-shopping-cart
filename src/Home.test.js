import Home from './Home';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

test('render correctly', () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <Home />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
