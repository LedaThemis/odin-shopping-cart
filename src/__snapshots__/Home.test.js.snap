// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<div
  className="Home"
>
  <h2
    className="welcome--header"
  >
    Welcome to Our Shop!
  </h2>
  <p>
    We Hope you find what you're looking for.
  </p>
</div>
`;
